testing/src/cypress/custom-command/institution/InstitutionCustomCommand.ts

```diff
@@ -1,25 +1,20 @@
 import LoginInstitutionObject from "../../page-objects/Institution-objects/LoginInstitutionObject";
 
 export default class InstitutionCustomCommand {
-  loginInstitution() {
+  loginWithCredentials(uname: string, pass: string) {
     const loginInstitutionObject = new LoginInstitutionObject();
 
-    const username = Cypress.env("bceid");
-    const password = Cypress.env("password");
-
-    cy.intercept("GET", "**/bceid-account").as("bceidAccount");
     loginInstitutionObject.loginWithBCEID().should("be.visible").click();
     loginInstitutionObject.loginInWithBCEIDtext().should("be.visible");
+
     loginInstitutionObject
       .bceidInputText()
-      .type(username)
-      .should("have.value", username);
+      .type(uname)
```

Comment:
just curious why `username` is updated to `uname`.  `username` is meaning and is the way that we are following

Reply:
same for `password`

Reply:
Sounds good. we can change that

Reply:
Done